src/GraphQLParser/ParserContext.Parse.cs

```diff
@@ -999,6 +999,7 @@ private GraphQLValue ParseObjectValue(bool isConstant)
 
         val.Comment = GetComment();
         val.Fields = ZeroOrMore(TokenKind.BRACE_L, isConstant ? constant : value, TokenKind.BRACE_R);
+        val.Value = _source.Slice(start, _currentToken.End - start - 1);
```

Comment:
Small fix here - `GraphQLListValue` had `Value` property, but `GraphQLObjectValue` did not.

Reply:
reverted